### ISSUE TITLE
Directname causes delay when switching between Pan and Orbit tools with the mouse

### ISSUE THREAD

**Issue by skylatian** [thomasa88/DirectName#12]
I have a very odd issue - when using pan/orbit tools with mouse middle/right click buttons, switching between the tools in a complex design has a significant delay. I enabled show pointer clicks in the attached screen recording - red is left click, green is left, middle is blue.

- It does not occur in simple designs
- It does not seem to occur with design history off
- It is not a performance issue, as moving with a Spacemouse performs perfectly fine
- It only occurs when DirectName is enabled

It's almost like DirectName is iterating through every component or something and checking if it needs to be named, any idea what may be going on here? I love the extension but it makes working with complex models a nightmare. Happy to provide any logs or test as needed, if possible!

https://github.com/user-attachments/assets/b2e451eb-7a2f-409b-b92e-f7964584b19a

**Comment by Scott216** [comment]
I don't see a file called DirectAdmin.py.   Did you mean DirectName.py?

**Comment by thomasa88** [comment]
Yes, exactly! My mistake :man_facepalming: 

I had some time to do a quick test now.
```
 Terminated command: ScriptsManagerCommand
 Terminated command: FreeOrbitCommand
 Terminated command: PanCommand
 Terminated command: PanCommand
 Terminated command: PanCommand
 Terminated command: FreeOrbitCommand
```
Looks like this:
![image](https://github.com/user-attachments/assets/4f7f6d9e-0743-462d-9067-3e2ad2f977ac)

So it looks like extending the list of ignored commands like this should help for pan and orbit:
```
    if args.commandId in ['SelectCommand', 'CommitCommand', 'PanCommand', 'FreeOrbitCommand']:
```

 but it is interesting to see if your navigation usage triggers some other command as well.

**Comment by thomasa88** [comment]
Hi, I just published [version 1.4.1 on Github](https://github.com/thomasa88/DirectName/releases/tag/v1.4.1), where I have added these and some more commands to the filtering. A new installer will hopefully appear in the Autodesk app store in 1-2 weeks.
